phoenix-hive/src/main/java/org/apache/phoenix/hive/PhoenixSerializer.java

```diff
@@ -64,12 +64,40 @@
 
     public PhoenixSerializer(Configuration config, Properties tbl) throws SerDeException {
         String mapping = tbl.getProperty(PhoenixStorageHandlerConstants.PHOENIX_COLUMN_MAPPING, null);
-        if(mapping!=null ) {
+        if(mapping != null) {
             config.set(PhoenixStorageHandlerConstants.PHOENIX_COLUMN_MAPPING, mapping);
         }
+
+        String tableName = tbl.getProperty(PhoenixStorageHandlerConstants.PHOENIX_TABLE_NAME);
```

Comment:
Could you add a comment as to why this change is needed for future readers? I'm guessing that Configuration is coming from the Hive framework, but the Properties is what the user actually provided to us?